content/en/glossary/terms/custom_span.md

```diff
@@ -3,4 +3,4 @@ title: custom span
 core_product:
   - ci-cd
 ---
-A custom span is the ability to integrate command-level events in CI Pipeline Visibility, which are displayed in a pipeline's flame graph visualization. For more information, <a href="/continuous_integration/pipelines/#supported-features">see the documentation</a>.
\ No newline at end of file
+A custom span is the ability to integrate command-level events in CI Pipeline Visibility, which are displayed in a pipeline's flame graph visualization. For more information, <a href="/continuous_integration/pipelines/custom_commands/">see the documentation</a>.
```

Comment:
Not related, but I saw that in the gloassary we were redirecting to the base page of the docs in CI Vis for `custom_spans`. We have a dedicated page in the docs for this so it might be better to redirect there.